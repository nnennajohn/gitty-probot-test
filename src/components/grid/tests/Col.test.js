// @flow
import React from 'react';
import renderer from 'react-test-renderer';

import { Col } from '../index';

test('Styles matches Col', () => {
  const tree = renderer.create(<Col />).toJSON();
  expect(tree).toMatchSnapshot();
  // expect(tree).toHaveStyleRule('box-sizing', 'border-box');
  // expect(tree).toHaveStyleRule('flex', '0 0 auto');
  // expect(tree).toHaveStyleRule('word-break', 'break-all');
});
