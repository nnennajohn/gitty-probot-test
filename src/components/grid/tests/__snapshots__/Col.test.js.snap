// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styles matches Col 1`] = `
.c0 {
  box-sizing: border-box;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  padding-right: 0.5rem;
  padding-left: 0.5rem;
  word-break: break-all;
}

@media only screen and (min-width:40em) {
  .c0 {
    padding-left: 0.5rem;
    padding-right: 0.5rem;
  }
}

<div
  className="c0"
/>
`;
